CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # Changelog
 
-## [1.6.5](../../releases/tag/v1.6.5) - Not released yet
+## [1.7.0](../../releases/tag/v1.7.0) - Not released yet
```

Comment:
In the case of an async client...

Since we are making additional requests, the `last_run` method must be made async. This change is technically breaking, although the original implementation was broken. Therefore, I have released it as a new minor version. This means it will not be updated automatically if you are using a tilde pin like `apify-client~=1.6.0`.

Reply:
Ok, then please mention it in the changelog too. I would still go with a patch release personally, especially since it was broken before anyway, but it's not a huge deal. I don't know how common it is to lock versions via `~`, but I am quite sure people do not do that to get around breaking changes.